### PR TITLE
Feature/migrate profile invitations

### PR DESCRIPTION
**Pull request by melisabok**
- Name deletion request using API 2 invitations
- Email deletion request using API 2 invitations
- Profile merge request using API 2 invitations

Fixes https://github.com/openreview/openreview-py/issues/1874

### CODE REVIEW

**Comment by xkopenreview** [comment]
the process log id is the entity.id which is the edit.id so currently the moderation page won't find matching log by comparing the log id with note id
@carlosmondra is it possible to add a note.id field?
another way is to get all the edit ids when query the email/name deletion notes

**Comment by melisabok** [comment]
> another way is to get all the edit ids when query the email/name deletion notes

what about querying the process logs by invitation instead of id?

**Comment by xkopenreview** [comment]
> > another way is to get all the edit ids when query the email/name deletion notes
> 
> what about querying the process logs by invitation instead of id?

process logs are already queried by invitation
but still need to find which log is for which note by comparing the log id

File: openreview/api/client.py [hunk 1]
```diff
@@ -588,6 +589,27 @@ def post_profile(self, profile):
...
+    def rename_profile(self, current_id, new_id):
+        """
+        Updates a Profile
+
+        :param profile: Profile object
+        :type profile: Profile
+
+        :return: The new updated Profile
+        :rtype: Profile
+        """
```
Comment (carlosmondra) [inline openreview/api/client.py:601]:
```suggestion
        """
        Updates a the profile id of a Profile

        :param current_id: Current profile id
        :type profile: str
        :param new_id: New profile id
        :type profile: str

        :return: The new updated Profile
        :rtype: Profile
        """
```

File: openreview/profile/management.py [hunk 2]
```diff
@@ -104,26 +104,38 @@ def set_remove_name_invitations(self):
...
+                        'optional': True
+                    }
+                }
             },
             'status': {
-                'order': 4,
-                'value-dropdown': ['Pending', 'Accepted', 'Rejected'],
-                'description': 'Request status.',
-                'required': True
+                'value': 'Pending'
```
Comment (carlosmondra) [inline openreview/profile/management.py:138]: Why was this changed to a constant value?
Is it OK to remove description and order from the invitation?

Comment (melisabok) [reply openreview/profile/management.py:138]: it is a constant value because there is another invitation that changes the status.

**Review by carlosmondra — APPROVED** [review]

File: openreview/profile/process/request_remove_name_process.py [hunk 3]
```diff
@@ -1,30 +1,31 @@
-def process(client, note, invitation):
+def process(client, edit, invitation):
 
     SUPPORT_USER_ID = ''
     REMOVAL_DECISION_INVITATION_ID = ''
-    baseurl_v2 = 'http://localhost:3001'
+    baseurl_v1 = 'http://localhost:3000'
+    print(edit.note)
```
Comment (xkopenreview) [inline openreview/profile/process/request_remove_name_process.py:6]: there's a print

File: openreview/profile/management.py [hunk 4]
```diff
@@ -236,20 +273,35 @@ def set_remove_email_invitations(self):
             'email': {
                 'order': 1,
                 'description': 'email that want to be removed.',
-                'value-regex': '.*',
-                'required': True                
+                'value': {
+                    'param': {
+                        'type': 'string',
+                        'regex': r"^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)"
```
Comment (xkopenreview) [inline openreview/profile/management.py:279]: i think this regex is incomplete

**Review by xkopenreview — APPROVED** [review]

**Review by celestemartinez — APPROVED** [review]
